frontend/src/pages/modelServing/useModelMetricsEnabled.ts

```diff
@@ -6,8 +6,10 @@ const useModelMetricsEnabled = (): [modelMetricsEnabled: boolean] => {
   ).status;
   const biasMetricsAreaAvailable = useIsAreaAvailable(SupportedArea.BIAS_METRICS).status;
 
+  const nimMetricsAreaAvailable = useIsAreaAvailable(SupportedArea.BIAS_METRICS).status;
```

Comment:
```suggestion
  const nimMetricsAreaAvailable = useIsAreaAvailable(SupportedArea.NIM_MODEL).status;
```

I think you intended to check `NIM_MODEL`?